app/components/article-featured-video-jwplayer.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember';
+import VideoLoader from '../modules/video-loader';
+import extend from '../utils/extend';
+
+const {
+	Component,
+	inject,
+	on,
+	observer,
+} = Ember;
+
+export default Component.extend({
+	ads: inject.service(),
+
+	// when navigating from one article to another with video, we need to destroy player and
+	// reinitialize it as component itself is not destroyed. Could be done with didUpdateAttrs
+	// hook, however it is fired twice with new attributes.
+	videoIdObserver: on('didInsertElement', observer('model.embed.jsParams.videoId', function () {
```

Comment:
Why can't we reinstantiate the whole component instead of cleaning the scope in the observer?